### ISSUE TITLE
false positives due to locale settings

### ISSUE THREAD

**Issue by dilbirligi** [IDPF/epubcheck#711]
It seems epubcheck is not handling page-lists as 
https://idpf.github.io/a11y-guidelines/content/nav/pagelist.html
says it should.
I get the following even though i don't think i should:
ERROR(OPF-027): OEBPS/nav.xhtml(91,41): Undefined property: 'page-list'.
ERROR(NAV-003): OEBPS/content.opf(-1,-1): The Navigation Document must have a page list when content document(s) contain page breaks (epub:type="pagebreak").

And all of the examples with page-lists that i've found (such as childrens-literature.epub) also fail with these kinds of errors. I even tried changing type to EDUPUB, but that doesn't improve things (& i think it shouldn't be required to have a page-list in EPUB3 anyway).

Strangely, idpf.org validation says no errors on the same epubs, & it says it also uses epubcheck 4.0.1.

**Comment by dilbirligi** [comment]
The following is my starter page-list (**nav.xhtml**):
```
			<nav epub:type="page-list" hidden="hidden"><!--  id="pages"-->
					 <h2>Pagebreaks of the print version, first edition</h2> 
					 <ol>
								<li>
										  <a href="title-page-i.xhtml#p01">1</a>
								</li>
								<li>
										  <a href="title-page-ii.xhtml#p02">2</a>
								</li>
					 </ol>
			</nav>
```
And the following is source (**content.opf**):
```
			<dc:source id="src">http://www.ebay.com/itm/ULKULERiM-/122208566685?hash=item1c7432bd9d:g:tOIAAOSwHMJYGO6N</dc:source>
		        <meta property="source-of" refines="#src">pagination</meta>
```
And the following is a sample page break (**title-page-i.xhtml**#p01):
`<div class="center"><span class="pb" epub:type="pagebreak" title="1" id="p01">1</span></div>`

P.S. So now i'm faced w/ a decision:
1. use this cause it works (e.g., in ADE; & hopefully epucheck wil eventually be fixed not to give 2 errors about page-list setup for no reason);
OR
2. don't use a page-list at all simply because buggy epubcheck doesn't like it, & i'm an obsessive dude who would like to have no errors or warnings from epubcheck.
P.P.S. What's your take on this, & how i should proceed?
P.P.P.S. I even adjusted (& thus resolved) everything additional related to EDUPUB, but the 2 errors mentioned epubcheck gives for a page-list regardless of type being EPUB or EDUPUB.

**Comment by dilbirligi** [comment]
Actually, after commenting out edupub (
`<!--		  <dc:type>edupub</dc:type>-->`
), i'm only seeing 1 error from epubcheck:
_ERROR(OPF-027): .../OEBPS/nav.xhtml(92,47): Undefined property: 'page-list'._

But it still makes me ponder over whether i should prefer using page-list feature or having no errors from epubcheck (choices in P.S. above).

**Comment by dilbirligi** [comment]
Getting the same in source code:
```
java -jar "C:\...\epubcheck\4.0.1\epubcheck.jar" "epubcheck\src\test\resources\com\adobe\epubcheck\test\xhtml\lorem_pagemaps1" --mode exp
Validating using EPUB version 3.0.1 rules.
ERROR(RSC-005): epubcheck\src\test\resources\com\adobe\epubcheck\test\xhtml\lorem_pagemaps1.epub/EPUB/lorem.opf(17,37): Error while parsing file 'attribute "page-map" not allowed here; expected attribute "id" or "page-progression-direction"'.
ERROR(OPF-027): epubcheck\src\test\resources\com\adobe\epubcheck\test\xhtml\lorem_pagemaps1.epub/EPUB/nav.xhtml(25,34): Undefined property: 'page-list'.

Check finished with errors

epubcheck completed
```

**Comment by rdeltour** [comment]
I can't reproduce here, and it was never reported before that `page-list` created issues (the property is definitely supported, we have some tests for it).

Let's try to troubleshoot the issue!

- what's your exact environment? (OS, Java version).
- where did you get EpubCheck? (latest release download from GitHub? or is it a custom build?)

**Comment by dilbirligi** [comment]
I have determined that the issue disappears after changing the default locale of my Windows 7 user from Turkish to English: which means there is some kind of dependency on locale in epubcheck's or java's functioning during page-list validation, which breaks it (and possibly other features). I have updated the issue title to reflect this finding.

P.S. I tried debugging epubcheck in a pristine setting: new user account, new workspace, new Eclipse project. To my surpise, it reported no errors on the same .epub. I was puzzled. I went back to the other account: tweaking PATH, JAVA_HOME, MVN_HOME, etc. had no effect. I was puzzled even more. There was no CLASSPATH set in either account. Eventually, the only difference between the 2 accounts that i could notice was that the original one was set up to use Turkish locale, and the new one was in English. After changing the original user account's locale from Turkish to English, epubcheck reported no errors on the same .epub (in cmd, & in cygwin):
```
No errors or warnings detected.
epubcheck completed
```
P.P.S. Hopefully you guys can fix this, cause i wasted about 2 days on this, and (most importantly) of course epubcheck should produce the same validation results (fatals, errors, warnings, etc.) regardless of which locale the current computer user is set up to use (the only acceptable differences being l10n-related ones).
